paddlenlp/seq2vec/encoder.py

```diff
@@ -145,34 +152,36 @@ def __init__(self,
             self._output_dim = maxpool_output_dim
 
     def get_input_dim(self):
-        """
+        r"""
         Returns the dimension of the vector input for each element in the sequence input
         to a `CNNEncoder`. This is not the shape of the input tensor, but the
         last element of that shape.
         """
         return self._emb_dim
 
     def get_output_dim(self):
-        """
+        r"""
         Returns the dimension of the final vector output by this `CNNEncoder`.  This is not
         the shape of the returned tensor, but the last element of that shape.
         """
         return self._output_dim
 
     def forward(self, inputs, mask=None):
-        """
+        r"""
         The combination of multiple convolution layers and max pooling layers.
 
         Args:
-            inputs (paddle.Tensor): Shape as `(batch_size, num_tokens, emb_dim)`
-            mask (obj: `paddle.Tensor`, optional, defaults to `None`): Shape same as `inputs`. 
-                Its each elements identify whether is padding token or not. 
+            inputs (`Tensor`): 
+                Shape as `(batch_size, num_tokens, emb_dim)`
+            mask (`Tensor`, optional): 
+                Shape shoule be same as `inputs`. Its each elements identify whether is padding token or not. 
```

Comment:
Its each elements identify whether is padding token or not.
->
Its each elements identify whether the corresponding input token is padding or not.

Reply:
done